rdrf/rdrf/forms/components.py

```diff
@@ -26,25 +26,30 @@ class LauncherError(Exception):
 
 class _Form:
 
-    def __init__(self, url, text, current=False, add_link_url=None, add_link_text=None):
+    def __init__(self, url, text, current=False):
         self.id = None
         self.url = url
         self.text = text
         self.current = current
-        self.add_link_url = add_link_url
-        self.add_link_text = add_link_text
         self.heading = ""
 
-        # Multiple contexts
+    def __str__(self):
+        return "Form %s %s %s" % (self.text, self.url, self.current)
+
+
+class _MultiContextForm(_Form):
+
+    def __init__(self, text, add_link_url, add_link_text):
+        super(_MultiContextForm, self).__init__(url=None, text=text, current=False)
```

Comment:
Usually we'd just call plain `super().__init__(...)` in this case

Reply:
I'll fix it 👍